.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,11 +11,6 @@
 - [ ] documentation changes
 - [ ] tested on mobile devices
 
-**Release notes:**
```

Comment:
Why do you removed this section? I thought that we generate release note document from this

Reply:
> I thought that we generate release note document from this
It looks the title of PR is used for generating release note. @niklabh Please correct me if I'm wrong.